<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vo_admin</class>
 <widget class="QMainWindow" name="vo_admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1251</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Панель администратора. Заказы</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../static/resource.qrc">
    <normaloff>:/img/img/Logotip.png</normaloff>:/img/img/Logotip.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1261</width>
      <height>151</height>
     </rect>
    </property>
    <widget class="QPushButton" name="btn_emp">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>401</width>
       <height>151</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: rgba(153, 102, 51);
    border: 2px solid rgba(100, 50, 51);
    color: white;
    font-weight: bold;
    font-size: 20pt;
}
QPushButton:hover {
    background-color: rgba(73, 140, 81);
}
QPushButton:pressed {
    background-color: rgba(73, 140, 81);
}
</string>
     </property>
     <property name="text">
      <string>Сотрудники</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>0</y>
       <width>461</width>
       <height>151</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: rgba(73, 140, 81);
    border: 2px solid rgba(43, 85, 48);
    color: white;
    font-weight: bold;
    font-size: 20pt;
}
</string>
     </property>
     <property name="text">
      <string>Заказы</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_smene">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>0</y>
       <width>391</width>
       <height>151</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: rgba(153, 102, 51);
    border: 2px solid rgba(100, 50, 51);
    color: white;
    font-weight: bold;
    font-size: 20pt;
}
QPushButton:hover {
    background-color: rgba(73, 140, 81);
}
QPushButton:pressed {
    background-color: rgba(73, 140, 81);
}
</string>
     </property>
     <property name="text">
      <string>Смены</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>1261</width>
      <height>491</height>
     </rect>
    </property>
    <widget class="QTableWidget" name="table_order">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>1231</width>
       <height>431</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid rgb(153, 102, 51);</string>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <column>
      <property name="text">
       <string>id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Официант</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Столик</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Количество клиентов</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Блюда</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Напитки</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Статус от официанта</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Статус от повара</string>
      </property>
     </column>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>440</y>
       <width>1261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="btn_auth">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>341</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: rgba(153, 102, 51);
    border: 2px solid rgba(100, 50, 51);
    border-radius: 20px;
    color: white;
    font-weight: bold;
    font-size: 12pt;
}
QPushButton:hover {
    background-color: rgba(73, 140, 81);
}
QPushButton:pressed {
    background-color: rgba(73, 140, 81);
}
</string>
      </property>
      <property name="text">
       <string>Выйти на форму авторизации</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../../static/resource.qrc"/>
 </resources>
 <connections/>
</ui>
